index.html

```diff
@@ -31,5 +31,142 @@ <h2>Let's Connect</h2>
+                        </ul>
+                    </div>
+                    <p class="body">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="stack">
+                        <li class="border-background">html</li>
+                        <li class="border-background">css</li>
+                        <li class="border-background">javascript</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+            </div>
+            <div class="display-card">
+                <div class="image-container">
+                    <img src="./assets/Snapshoot Portfolio (1).png" alt="Snapshoot Portfolio 1">
+                </div>
+                <div class="text-container">
+                    <h1>Tonic</h1>
+                    <div class="inline-text">
+                        <h4>Canopy</h4>
+                        <ul class="inline-list">
+                            <li>Back End Dev</li>
+                            <li>2015</li>
+                        </ul>
+                    </div>
+                    <p class="body">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="stack">
+                        <li class="border-background">html</li>
+                        <li class="border-background">css</li>
+                        <li class="border-background">javascript</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+            </div>
+            <div class="display-card">
+                <div class="image-container">
+                    <img src="./assets/Snapshoot Portfolio (2).png" alt="Snapshoot Portfolio 2">
+                </div>
+                <div class="text-container">
+                    <h1>Tonic</h1>
+                    <div class="inline-text">
+                        <h4>Canopy</h4>
+                        <ul class="inline-list">
+                            <li>Back End Dev</li>
+                            <li>2015</li>
+                        </ul>
+                    </div>
+                    <p class="body">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="stack">
+                        <li class="border-background">html</li>
+                        <li class="border-background">css</li>
+                        <li class="border-background">javascript</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+            </div>
+            <div class="display-card">
+                <div class="image-container">
+                    <img src="./assets/Snapshoot Portfolio (3).png" alt="Snapshoot Portfolio 3">
+                </div>
+                <div class="text-container">
+                    <h1>Tonic</h1>
```

Comment:
- [ ] Please 🙏🏽 correct the title of the second and fourth card to match the Figma template. It is "Multi-Post Stories"